### PR TITLE
Book Store Styling.

### PR DESCRIPTION
**Pull request by Alaska01**
In Milestone 5, the app made use of scss styling and there was an addition of statics components which had no functionality among others requirement.

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+.navbar {
+  width: 100%;
+  height: 95px;
+  background-color: #fff;
+  margin: 0 auto;
+
+  &.content {
+    max-width: 1100px;
+    border-radius: 4px;
+    display: flex;
```
Comment (rindrajosia) [inline src/App.scss]: - [ ] [REQUIRED] Please, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule

File: src/App.scss [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+  }
+}
+
+.book {
+  &.main-container {
+    width: 50%;
+  }
+
+  &.main {
+    display: flex;
```
Comment (rindrajosia) [inline src/App.scss]: - [ ] [REQUIRED] Please, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule

File: src/App.scss [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+
+  &.title {
+    height: 29px;
+    margin: 3px 323px 0 0;
+    font-size: 22px;
+    font-family: 'Roboto Slab', serif;
+    font-weight: bold;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
```
Comment (rindrajosia) [inline src/App.scss:92]: - [ ] [REQUIRED] Please, make sure that basic CSS bad practices are not used ([article for support](https://speckyboy.com/good-bad-css-practices) ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
- no repeating of the same rule

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:

## To highlight:

- I was able to add book :+1: 
- I was able to remove book :100: 
- I was able to filter book :1st_place_medal: 


## Required Changes ♻️

- Styling should be **EXACTLY** the same as the one provided in the designs ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone5/exactlySameAsDesign.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/45791599/))

(_The original design_)


- [ ] It will be nice if the avatar is aligned on the right (:arrow_double_down: ).
- [ ] Please align the book description on the left (:arrow_double_down: ).




_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.scss (hunk 1 above)
Comment (taiwo2) [reply src/App.scss]: KIndly implement these suggested changes by the previous code reviewer, you can make a class for `display: 'flex' `and apply it on all parts where `display: 'flex'` is used

**Review by taiwo2 — CHANGES_REQUESTED** [review]
Hi @Alaska01 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

Kindly check the design styling uploaded by the previous code reviewer to update your current app version. the avatar should align with the box
Screenshot 2021-11-12 191610

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alaska01** [comment]
There is currently no repetition of class names in the styling and the avatar position was worked on. You may wish to check and give another review. Thanks!!! A class was also created for the flex and re-used subsequently.

**Comment by Alaska01** [comment]
> Hi @Alaska01 ,
> 
> Great work on making the changes requested by a previous reviewer clap You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## Required Changes 
> _Check the comments under the review._
> 
> Kindly check the design styling uploaded by the previous code reviewer to update your current app version. the avatar should align with the box Screenshot 2021-11-12 191610
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Here is how I have repositioned the avatar. A screenshot here [https://drive.google.com/file/d/1sa1jYEsDS8pZfhA5imBugb0bU95UwVMB/view?usp=sharing](url)

**Comment by Alaska01** [comment]
A screenshot of how I repositioned the avatar. https://drive.google.com/file/d/1sa1jYEsDS8pZfhA5imBugb0bU95UwVMB/view?usp=sharing

File: src/components/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+  // The main-1 class was initially main, it may not be needed
+  return (
+    <div className="navbar main-1">
+      <div className="navbar content">
+        <h1>BookStore CMS</h1>
+        <h3>BOOKS</h3>
+        <h3>CATEGORIES</h3>
+        <CategoryFilter handleFilterChange={handleFilterChange} />
+        <span><i className="fas fa-user-alt" /></span>
```
Comment (taiwo2) [inline src/components/NavBar.js:19]: The Avatar should be on the same line with the button `ADD BOOK`. kindly implement it, I have tested it with three different browser gotten the same result

**Review by taiwo2 — CHANGES_REQUESTED** [review]
Hi @Alaska01 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
 ## Highlights #
no repeating of the same rule 👍🏻 

 ## Required Changes ♻️
_Check the comments under the review._

please kindly implement the design styling for the Avatar I still got the same thing
Screenshot 2021-11-12 204857

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by taiwo2** [comment]
> There is currently no repetition of class names in the styling and the avatar position was worked on. You may wish to check and give another review. Thanks!!! A class was also created for the flex and re-used subsequently.

the Avatar positioning is not working yet,I tested it with different browser I got the previous same result

**Comment by taiwo2** [comment]
> There is currently no repetition of class names in the styling and the avatar position was worked on. You may wish to check and give another review. Thanks!!! A class was also created for the flex and re-used subsequently.

Great job is done   for the class been made 👍🏻 reuseable

**Comment by Alaska01** [comment]
Had a zoom call with the previous reviewer and all exchanged screens to see how the positioning is working on various systems. The avatar is perfectly positioned on mine while not aligning on his system. I have adjusted the positioning again. I hope this position displays and fits well on your system, I would be more than happy if it displays well at your end. Enjoy and happy reviewing.

**Review by rukundoeric — APPROVED** [review]
Hi @Alaska01,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**